lib/design/app_theme.dart

```diff
@@ -0,0 +1,61 @@
+
+import 'package:flutter/material.dart';
+
+import 'my_colors.dart';
+
+class AppTheme {
+
+  final BorderRadius buttonBorderRadius = BorderRadius.circular(18.0);
+  final EdgeInsets mainButtonPadding = EdgeInsets.all(15);
```

Comment:
NIT: I like to define numbers as factor of 8 and as doubles so maybe change these to 16.0 but it doesn't really matter.